go/vt/vttablet/endtoend/config_test.go

```diff
@@ -219,6 +219,33 @@ func TestMaxResultSize(t *testing.T) {
 	}
 }
 
+func TestWarnResultSize(t *testing.T) {
+	defer framework.Server.SetWarnResultSize(framework.Server.WarnResultSize())
+	framework.Server.SetWarnResultSize(2)
+	client := framework.NewClient()
+
+	originalWarningsResultsExceededCount := tabletenv.Warnings.Counts()["ResultsExceeded"]
```

Comment:
Could you instead use framework.DebugVars? Here's an example: https://github.com/youtube/vitess/blob/master/go/vt/vttablet/endtoend/misc_test.go#L195.